src/shared/constants.h

```diff
@@ -88,6 +88,7 @@ static const double NANOSECONDS_PER_SECOND       = 1000000000.0;
 static const double MICROSECONDS_PER_MILLISECOND = 1000.0;
 static const double MICROSECONDS_PER_SECOND      = 1000000.0;
 static const double MILLISECONDS_PER_SECOND      = 1000.0;
+static const double CHICKER_TIMEOUT              = 3 * MILLISECONDS_PER_SECOND;
```

Comment:
minor nit: It would make a bit more sense with the organization of this file to move this constant somewhere else. Potentially lower down to here (similarly for the pybind): https://github.com/UBC-Thunderbots/Software/blob/8ebbece83013c0c2ccac434090d457e85c66106d/src/shared/constants.h#L119